src/ainftBase.ts

```diff
@@ -0,0 +1,31 @@
+import Ain from "@ainblockchain/ain-js";
+import { AINFT_SERVER_ENDPOINT } from "./constants";
+import stringify = require("fast-json-stable-stringify");
+
+export default class AinftBase {
+  public baseUrl: string;
+  public route: string;
+  public ain: Ain;
+
+  constructor(
+    baseUrl = AINFT_SERVER_ENDPOINT,
```

Comment:
minor) what about moving this default parameter after required parameters so that can be easily omitted or just changing it to a required parameter?

Reply:
Done! moved after required parameter.